bindings_node/src/groups.rs

```diff
@@ -469,6 +469,37 @@ impl NapiGroup {
     Ok(group_name)
   }
 
+  #[napi]
+  pub async fn update_group_image_url_square(&self, group_image_url_square: String) -> Result<()> {
+    let group = MlsGroup::new(
+      self.inner_client.context().clone(),
+      self.group_id.clone(),
+      self.created_at_ns,
+    );
```

Comment:
Would it be better to: rather than pass the whole client just to get a group with `group_id` instantiated, to instead put this function (as a method) on the `MLSGroup` struct directly?

Reply:
i'm not sure i understand this question

Reply:
nvm.  Looking at this PR more closely, I see you're converting over from a `NapiGroup` to an `MLSGroup` to call the method; passing through.  I mistakenly thought this is being implemented on the client.

Please proceed with merging 😆